src/callbacks/steady_state.jl

```diff
@@ -0,0 +1,54 @@
+
+"""
+    SteadyStateCallback(; abstol=1.0e-8, reltol=1.0e-6)
+
+Terminates the integration when the [`residual_steady_state(du, equations)`](@ref)
+falls below the threshold specified by `abstol, reltol`.
+"""
+mutable struct SteadyStateCallback{RealT<:Real}
+  abstol::RealT
+  reltol::RealT
+end
+
+function SteadyStateCallback(; abstol=1.0e-8, reltol=1.0e-6)
```

Comment:
Are we sure that default tolerances make sense here? I am really asking, maybe they do, but intuitively I would say that this is so application dependent it does not make much sense (and that instead we should force the users to pick values.